### PR TITLE
New Privacy Settings section

### LINKED ISSUE #9359: Add privacy settings section
We have a new design for the tracks opt out that offers more information to users about their privacy

mobile-privacy-blueprint-i2

### PR DESCRIPTION
**Pull request by koke**
Replaces the analytics opt-out with a dedicated privacy page with more
information and links to the privacy and cookies policies.

@frosty besides a review, I need a bit of help with the UI. According to @folletto's design, the icons should be top-aligned, and the read more links should be left-aligned with the other texts. I'm not sure how to do either, and you might have some ideas 🙏 

simulator screen shot - iphone x - 2018-05-14 at 19 10 16

Fixes #9359 

To test:

- Go to App Settings > Privacy Settings
- Verify that it matches the design in #9359 
- Make sure the toggle works

### CODE REVIEW

**Comment by frosty** [comment]
So for the `PaddedLinkRow` indentation, I think the simplest option might be to add an empty placeholder image in `configureCell(_:)`:

`cell.imageView?.image = UIImage(color: .clear, havingSize: Gridicon.defaultSize)`

**Comment by frosty** [comment]
I'm not entirely sure the best way to align the icons. I think you'll need a custom cell. You could either just manually move the image view around in `layoutSubviews` (probably not the best), or add a custom image view and add some constraints to the text. The easiest constraint would just be pinning to the top of the cell or the text's first baseline with an offset. Pinning it to the center of the first line of text is a little tricky though. Off the top of my head, but it feels like a hack:

* Add a second label behind the main label, matching the same font. You might need to add a single empty space to it so that it has a size...
* Pin your custom image view's center Y to the center Y of the label
* The second label is needed so it's always one line tall – the normal label wraps over multiple lines.
* The custom image view is needed because the system will try and reposition the standard `imageView`.

File: WordPress/Classes/ViewRelated/Me/PrivacySettingsViewController.swift [hunk 1]
```diff
@@ -0,0 +1,182 @@
+import Gridicons
+import UIKit
+
+class PrivacySettingsViewController: UITableViewController {
+    fileprivate var handler: ImmuTableViewHandler!
+
+    override init(style: UITableViewStyle) {
+        super.init(style: style)
+        navigationItem.title = NSLocalizedString("App Settings", comment: "App Settings Title")
```
Comment (frosty) [inline WordPress/Classes/ViewRelated/Me/PrivacySettingsViewController.swift]: Title should be 'Privacy Settings', not 'App Settings'.

**Comment by koke** [comment]
> The icons in the info rows look slightly too low to me

They are, but not sure how to nail that down. I've used the `layoutMargins` reference, but it's 2px lower than the switch row

screen shot 2018-05-15 at 12 38 05

> On the original designs you linked to, the icons look like they're darkGrey. Should we match that tint color?

That's right, but all the rest of icons in the app seem to have this tint

**Comment by folletto** [comment]
> all the rest of icons in the app seem to have this tint

Yep: note that even the "previous" screen was redesigned, so you can adjust to keep consistency with the existing structure. I've still done the design as a full redesign including the previous page for future consistency. :)

**Comment by frosty** [comment]
@koke Hmm, I don't know of a nice way to fix that. You could just minus an offset from the `origin.y`, or I think it's a case of a custom cell.

**Comment by koke** [comment]
About the icon colors, I believe it should be a quick change in the style guide, but since it affects many different screens in the app, I'd handle it in a different PR

**Comment by frosty** [comment]
@koke I agree with that. Shall we just offset the icons manually by 2px for now, and :shipit:?

**Comment by frosty** [comment]
I've pushed a small tweak for the icon positioning. Let's get this merged!

**Comment by frosty** [comment]
@koke I had one more attempt, and it's better. I think I've spent enough time on this now, so unless you have any major objections, let's :shipit:?

At default text size, this now matches the spacing on a default cell:

screen shot 2018-05-17 at 18 12 14

At larger sizes it's okay too:

screen shot 2018-05-17 at 18 12 39

At very larger "larger text" accessibility sizes it's not correct, but it sticks to the top layout guide of the cell. I think this acceptable?

screen shot 2018-05-17 at 18 20 03

**Pull request merged**